### PR TITLE
Add amount discount on product thumbnails

### PR DESCRIPTION
**Pull request by fatmaBouchekoua**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Discounts shown on thumbnails for percentage, and not for amount
| Type?         | improvement
| Category?     | FO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-3030
| How to test?  | Add a specific price to a product, an amount discount, and visualize the product on FO.

### CODE REVIEW

**Comment by vincentbz** [comment]
Hi @fatmaBouchekoua 

Thanks for that PR, the discount is now shown.
But behind the discount, the product name is also shown 😄 is it possible to remove it please ? 

image
image

**Comment by fatmaBouchekoua** [comment]
Hello @vincentbz ,
The product name is shown because this product doesn't have image ( the alt attribute of the img tag)

File: themes/classic/templates/catalog/_partials/miniatures/product.tpl [hunk 1]
```diff
@@ -49,7 +49,9 @@
                 <span class="sr-only">{l s='Regular price' d='Shop.Theme.Catalog'}</span>
                 <span class="regular-price">{$product.regular_price}</span>
                 {if $product.discount_type === 'percentage'}
-                  <span class="discount-percentage">{$product.discount_percentage}</span>
+                  <span class="discount-percentage discount-product">{$product.discount_percentage}</span>
+                {elseif $product.discount_type === 'amount'}
+                  <span class="discount-amount discount-product">-{$product.discount_amount}</span>
```
Comment (maximebiloe) [inline themes/classic/templates/catalog/_partials/miniatures/product.tpl]: Could you please remove this "calculation" from the template to do it in the presenter like we do it for the percentage discount here: https://github.com/PrestaShop/PrestaShop/blob/develop/src/Core/Product/ProductPresenter.php#L159

**Review by maximebiloe — CHANGES_REQUESTED** [review]

File: themes/classic/templates/catalog/_partials/miniatures/product.tpl (hunk 1 above)
Comment (fatmaBouchekoua) [reply themes/classic/templates/catalog/_partials/miniatures/product.tpl]: Done Thanks !

**Review by maximebiloe — DISMISSED** [review]

**Comment by maximebiloe** [comment]
Hello @fatmaBouchekoua,

Could you rebase you PR and make the changes on the `StarterTheme` too, please?

Regards

**Review by mickaelandrieu — APPROVED** [review]

**Comment by Quetzacoalt91** [comment]
Thank you @fatmaBouchekoua

**Pull request merged**